University-advisor-web/Models/UserModel.cs

```diff
@@ -15,5 +14,24 @@ public class UserModel
         public string University { get; set; }
         public string Status { get; set; }
         public string Password { get; set; }
+        public string CurrentPassword { get; set; }
+        public string NewPassword { get; set; }
+        public string NewPassword2 { get; set; }
+
+        public UserModel() { }
```

Comment:
Empty constructor goes by default, it is useless here.

Reply:
No, it requires an empty constructor.